src/Sylius/Behat/Page/Shop/Account/DashboardPage.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Account;
+
+use Sylius\Behat\Page\SymfonyPage;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowksi@lakion.com>
+ */
+class DashboardPage extends SymfonyPage implements DashboardPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
```

Comment:
In my opinion, this is a public method and it in some way defines the current page, so it should be as a first one.